lms/static/js/student_account/views/InstitutionLoginView.js

```diff
@@ -1,30 +1,28 @@
-var edx = edx || {};
-
-(function($, _, Backbone) {
+;(function (define) {
     'use strict';
+    define(['jquery', 'underscore', 'backbone'], function($, _, Backbone) {
 
-    edx.student = edx.student || {};
-    edx.student.account = edx.student.account || {};
-
-    edx.student.account.InstitutionLoginView = Backbone.View.extend({
-        el: '#institution_login-form',
+        var InstitutionLoginView = Backbone.View.extend({
```

Comment:
Instead of assigning the view to a variable and returning that you can just return the view, i.e.
```
return Backbone.View.extend({
```